src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs

```diff
@@ -0,0 +1,65 @@
+// <copyright file="OpenTelemetryLoggerProvider.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NETSTANDARD2_0
+using System.Collections.Concurrent;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace OpenTelemetry.Logs
+{
+    [ProviderAlias("OpenTelemetry")]
+    public class OpenTelemetryLoggerProvider : ILoggerProvider, ISupportExternalScope
+    {
+        private readonly IOptionsMonitor<OpenTelemetryLoggerOptions> options;
+        private readonly ConcurrentDictionary<string, OpenTelemetryLogger> loggers;
+        private IExternalScopeProvider scopeProvider;
+
+        public OpenTelemetryLoggerProvider(IOptionsMonitor<OpenTelemetryLoggerOptions> options)
+        {
+            this.options = options;
+            this.loggers = new ConcurrentDictionary<string, OpenTelemetryLogger>();
+        }
+
+        internal IExternalScopeProvider ScopeProvider
+        {
+            get
+            {
+                if (this.scopeProvider == null)
+                {
+                    this.scopeProvider = new LoggerExternalScopeProvider();
+                }
+
+                return this.scopeProvider;
+            }
+        }
+
+        void ISupportExternalScope.SetScopeProvider(IExternalScopeProvider scopeProvider)
+        {
+            this.scopeProvider = scopeProvider;
+        }
+
+        public ILogger CreateLogger(string name)
```

Comment:
On the ILoggerProvider interface param is actually `categoryName` I think (going from memory), might want to rename here and on OpenTelemetryLogger ctor?

Reply:
Let me look into this, I was following the dotnet runtime code https://github.com/dotnet/runtime/blob/932cc5e7e38098c75aba0596b6375a0ddba8a0d8/src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs#L124.

Reply:
You're right @CodeBlanch, it is indeed categoryName according to the [official doc](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.iloggerprovider.createlogger#parameters). Thanks for the catch! 👍